samples/javascript_typescript/12.nlp-with-luis/README.md

```diff
@@ -66,21 +63,19 @@ NOTE: Once you publish your app on LUIS portal for the first time, it may take s
 Language Understanding service (LUIS) allows your application to understand what a person wants in their own words. LUIS uses machine learning to allow developers to build applications that can receive user input in natural language and extract meaning from it.
 
 # Deploy this bot to Azure
-You can use the [MSBot](https://github.com/microsoft/botbuilder-tools) Bot Builder CLI tool to clone and configure any services this sample depends on.
-
-To install all Bot Builder tools -
+You can use the [MSBot](https://github.com/microsoft/botbuilder-tools) Bot Builder CLI tool to clone and configure any services this sample depends on. Ensure you have [Node.js](https://nodejs.org/) version 8.5 or higher.
 
-Ensure you have [Node.js](https://nodejs.org/) version 8.5 or higher
+To install all Bot Builder tools
```

Comment:
Instead of this line, you could add a reference to `Installing CLI tools` file as done [here](https://github.com/southworkscom/BotBuilder-Samples/pull/66/files#diff-121cd7b2bed5cb006c98e3741f97a145R26)

Reply:
@dfavretto Reference added